pegasus/sites.v3/code.org/public/challenge-new.md.partial

```diff
@@ -2,100 +2,86 @@
+Post or submit your creations for Band and Code.org to see, and we'll share the best ones on social media. Plus, you'll get a chance to win tickets to see Band on tour, or a chance for your classroom to video chat with the band.
 
-We can’t wait to see and share what you create! 
+We can't wait to see and share what you create! 
 
 
-## How to join the fun — code, dance, and share!
-<ol>
-<li>Start <a href="https://code.org/dance">Dance Party</a></li>
-<li>Learn to code your own dance in a short 10-step tutorial</li>
-<li>On the final level, code an awesome dance to Band's song, “Song.” (Don’t forget to try out the special “Song” background effect!)</li>
-<li>Click “Finish” to get a share link to your project.</li>
-<li>For extra fun, make a video of yourself dancing along with your code!</li>
-</ol>
+## How to join the fun - code, dance, and share!
+1. Start [Dance Party](https://code.org/dance)
+2. Learn to code your own dance in a short 10-step tutorial
+3. On the final level, code an awesome dance to Band's song, "Song." (Don't forget to try out the special "Song" background effect!)</li>
+4. Click "Finish" to get a share link to your project.
+5. For extra fun, make a video of yourself dancing along with your code!
 
 ## Share your creations for Code.org and Band to see!
 
-We’d love to see what classrooms and students create — whether it’s dance moves in code, or a video of yourself dancing along. Band and Code.org will re-share or comment on the most creative stuff we see. There are two ways to share:
-<p>
+We'd love to see what classrooms and students create - whether it's dance moves in code, or a video of yourself dancing along. Band and Code.org will re-share or comment on the most creative stuff we see. There are two ways to share:
+
 ### Post on social media:
-<a href="#howtowin"><img src="/images/marketing/promo/entry_graphic2.png" style="max-width: 50%"></a>
-<ol><strong>
-<li>Tell us your concert city. <a href="https://www.coldplay.com/tour/">See list of tour cities here.</a> Available tickets are for the first night of each city.</li>
-<li>Tag Code.org and use #codeplay</li>
-<li>Show us your "Song" Dance Party creation!</li></strong>
-</ol>
-<p>
-<em>If you post to social media, make sure to follow Code.org so we can message you:  <a href="https://twitter.com/codeorg">Twitter</a> | <a href="https://www.instagram.com/codeorg/">Instagram</a> | <a href="https://www.tiktok.com/@codeorg">TikTok</a> | <a href="https://www.facebook.com/Code.org/">Facebook</a></em>
-
-<b>Using TikTok or Instagram?</b> Instead of sharing a link to your project, share a video of it! Dance along with your creation — you can even try getting creative with TikTok’s Duets feature.
-
-<p>
+[<img src="/images/marketing/promo/entry_graphic2.png" style="max-width: 50%">](#howtowin)
```

Comment:
Can you use the markdown syntax for images rather than an <img/> tag? I think this will expose the HTML to the translators (assuming we want this page translated).

Reply:
I did this so that I could include the CSS styling as well.  From what I can tell, we do allow HTML to be translated, even if we don't prefer it?